src/pages/edge/lifecycle-for-edge-network/metrics.md

```diff
@@ -33,6 +33,23 @@ The following data is collected on each [Lifecycle Application Foreground](../..
 | xdm:isUpgrade | boolean | Indicates the application was upgraded. If `true`, signifies the first launch of the application after an upgrade. |
 | dc:language | String | The language of the application to represent the user's linguistic, geographical, or cultural preferences for data presentation. |
 
+<InlineAlert variant="neutral" slots="header, text1, text2, text3, text4, text5" />
+
+NOTE
+
+For the [Edge Network extension](../edge-network/index.md), the `xdm:isUpgrade` property, which identifies app upgrades, is collected within the [Lifecycle extension](../../home/base/mobile-core/lifecycle/) by comparing version information at each app launch. Recently, there is an enhancement to the method of calculating this property.
+
+**xdm:version**
+This property captures the version in the format of `<version string> (<version code>)` on Android and `<version string> (<build number>)` on iOS.
+
+**xdm:isUpgrade (Previous Method)**
+In Lifecycle extension for Android version (2.0.0 - 3.0.0) and Lifecycle extension for iOS version (4.0.0 - 5.0.0), the app upgrade event is detected by comparing the `build number`s on iOS and the `version string`s on Android.
+
+**xdm:isUpgrade (New Method)**
+Starting with Lifecycle extension for Android version 3.0.1 and Lifecycle extension for iOS version 5.0.1, the app upgrade event is detected through comparisions of the `xdm:version` properties.
+
+This enhancement is designed to seamlessly integrate with existing installations, as the mobile app's `version string` and `version code` typically change together with each upgrade. So, this enhancement is expected to be non-disruptive for current SDK users.
```

Comment:
Instead of this, maybe state explicitly how this might affect some customers. Some like:

This change now more accurately detects when an application upgrade occurs. Applications which only change the `build number` on Android or the `version string` on iOS when upgrading, may see an increase in Lifecycle upgrade events. Other applications, however, should not see a change to Lifecycle upgrade events.

Reply:
@yangyansong-adbe What do you think of this suggestion? Can you update the wording, or if you have other suggestions let me know.